src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @Size(min = 4)
```

Comment:
How about movies with title length < 4, e. g. "It"? I'd recommend you to remove constraints from movie titles.